### DISCUSSION TITLE
Architects Daughter is not the default anymore for Sketch diagram?

### DISCUSSION THREAD

**Discussion by MrDesjardins** [jgraph/drawio#4216] [Q&A] [answered]
Since last week, I have noticed that all my new Draw.io diagrams, using the Sketch style, use Helvetica font instead of the Architects Daughter. 

I can add it manually with Google Fonts. However, I must manually set the font to every element I drag whenever I create a new diagram. It is additional work, and I am wondering if I might have clicked something somewhere without noticing. 

Is there a way to set it back to at least keep the default font automatically applied to the element we drag?

**Comment by alderg** [comment]
If you are using a configuration for dafaultVertexStyle or defaultEdgeStyle, try adding this:
```
"fontFamily": "Architects Daughter",
"fontSource": "https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DArchitects%2BDaughter"
```

**Reply by MrDesjardins** [reply]
@alderg  I'm not sure where are these configurations. I added the font in `Text>Font>Custom...>Google Fonts`.
My problem is that every time I'm dragging from the tool box an item that has text that I must select the font manually. In the past (prior to February 2024) once the "Sketch" style was selected, every thing had the Architects Daughter font. No need to set once, neither need to set for every item we were dragging.

Why was there a change? It does not seem to be practical to set font on every single item.

**Reply by davidjgraph** [reply]
Extras->Configurations. There was no change.

**Reply by MrDesjardins** [reply]
@davidjgraph Very odd. I'm using Draw.io every week since months and I never had to set the font from Google before or adding a configuration 

Okay, so, I copied: 

```
{
  "fontFamily": "Architects Daughter",
  "fontSource": "https%3A%2F%2Ffonts.googleapis.com%2Fcss%3Ffamily%3DArchitects%2BDaughter"
}
```

into the `Extras->Configuration `and I restarted the app (as mentioned when saving the configuration). Still the font is not there without manually adding it. 

https://github.com/jgraph/drawio/assets/1014796/391778a9-a78b-4b57-afad-9d0d8c47da1b

**Reply by MrDesjardins** [reply]
If I go through the UI (Font>Custom) and add the font name. I see in the Extras>Configuration>Preferences:

```
 "customFonts": [
    {
      "name": "Architects Daughter",
      "url": "https://fonts.googleapis.com/css?family=Architects+Daughter"
    }
  ],
```
While this add the possibility to select manually the font for each individual item it does not make it the default font.

**Comment by alderg** [comment]
What version do you see in the help menu? If you open a new incognito window (Ctrl+Shift+N in Chrome on Windows), go to sketch.diagrams.net, add a rectangle and type something, what font do you see?

**Comment by MrDesjardins** [comment]
> What version do you see in the help menu? If you open a new incognito window (Ctrl+Shift+N in Chrome on Windows), go to sketch.diagrams.net, add a rectangle and type something, what font do you see?

I see Architects Daughter!

**Reply by MrDesjardins** [reply]
Does that mean that http://sketch.diagrams.net is the new way to go?

**Comment by alderg** [comment]
This is a bug. It will be fixed in the next release.

**Reply by davidjgraph** [reply, marked as answer]
Fix in 24.0.3.

**Reply by MrDesjardins** [reply]
@alderg  @davidjgraph Thanks!

**Reply by MrDesjardins** [reply]
Just to let you know that it came back since few weeks. I had the custom Google Font and it does not work anymore. I am using Sketch.diagrams.net which work.

**Comment by alderg** [comment]
You mention sketch.diagrams.net works fine — are you using the desktop app or app.diagrams.net where it's broken?

Could you also check what you have in Extras > Configuration there? If you have `defaultVertexStyle`, `defaultEdgeStyle`, or `defaultFonts` set there, that overrides the built-in sketch defaults (including the Architects Daughter font). The earlier workaround with `fontFamily`/`fontSource` in the config may actually be interfering now.

Try clearing your configuration (Extras > Configuration, remove everything or set it to `{}`), then restart. The sketch theme should apply Architects Daughter automatically without any custom config.
